docker-versions-maven-plugin/src/main/java/com/github/cafapi/docker_versions/plugins/extension/DockerVersionsLifecycleParticipant.java

```diff
@@ -176,4 +181,46 @@ private static void printBuildOrder(final List<MavenProject> projects)
         LOGGER.debug("--- Build order of {} projects --- ", totalModules);
         projects.stream().forEach(p -> LOGGER.debug("{}", p.getName()));
     }
+
+    private static void setProjectDockerRegistryProperty(final MavenSession session)
+    {
+        // TODO: Throw an exception if the property is already set
```

Comment:
- Have the extension throw an error if the property is set in the pom.
- Undecided: Do we let users override at the command line?